requirements.txt

```diff
@@ -21,6 +21,6 @@ tensorboard>=1.15.0
 tqdm>=4.30
 scipy
 torchio>=0.18.68
-torch>=1.8.1,<=1.11.0
+torch==2.0.0
```

Comment:
Some initial observations:

- Python 3.7 support stops at `torch==1.13.1`. As far as SCT goes, this is okay, but as far as ivadomed maintenance goes, updating to `torch==2.0.0` means removing support for Python 3.7.
- `torchvision` and `torchio` versions are tied to `torch` versions. So, we'll also have to bump the upper bound of those packages, too.